src/ocamlorg_frontend/layouts/learn_layout.eml

```diff
@@ -111,13 +111,11 @@ inner_html
              id="htmx-content">
           <%s! inner_html %>
         </div>
-        <% (match right_sidebar_html with
-            | Some (html) -> %>
-          <div class="hidden xl:flex top-0 sticky h-screen flex-col w-60 overflow-auto lg:pt-6 right-sidebar"
-               id="htmx-right-sidebar">
-            <%s! html %>
-          </div>
-        <% | None -> () ); %>
+        <div class="hidden <%s if Option.is_some right_sidebar_html then "xl:flex top-0 sticky h-screen flex-col w-60 overflow-auto lg:pt-6 right-sidebar" else "" %>"
```

Comment:
```suggestion
        <div class="hidden <%s Option.fold right_sidebar_html ~none:"" ~some:(Fun.const "xl:flex top-0 sticky h-screen flex-col w-60 overflow-auto lg:pt-6 right-sidebar") %>"
```

I wonder if we should have specialized versions of `Option.fold` and `Option.value` on `string option` and with `""` as default? How often do we return `""` on `string option` throughout `.eml` files?
 
```ocaml
        <div class="hidden <%s Html.maybe right_sidebar_html ~html:(Fun.const "xl:flex top-0 sticky h-screen flex-col w-60 overflow-auto lg:pt-6 right-sidebar") %>"
```